lsp-protocol.el

```diff
@@ -290,6 +290,9 @@ See `-let' for a description of the destructuring mechanism."
                (rust-analyzer:CommandLink (:title :command) (:arguments :tooltip))
                (rust-analyzer:CommandLinkGroup (:commands) (:title)))
 
+(lsp-interface
+ (PrepareRenameDefaultBehaviorResult (:defaultBehavior) nil))
+
```

Comment:
You should add it on line 596, not here.

Reply:
That is after ";;; begin automatically generated code". Wouldn't this be overwritten by some script?

Reply:
No, this is not going to be regenerated anymore. We are supposed to add the new fields manually.

Reply:
seems like I have misled you here. We should generate a new type, not alter the existing one. I thought that they are adding a new field but they are actually creating a new result type.